### PR TITLE
folder rename breadcrumb update fix

### PR DESCRIPTION
**Pull request by jdaev**
[Issue](http://redmine.ardrive.io/issues/206)

### CODE REVIEW

File: lib/blocs/fs_entry_rename/fs_entry_rename_cubit.dart [hunk 1]
```diff
@@ -21,7 +21,7 @@ class FsEntryRenameCubit extends Cubit<FsEntryRenameState> {
   final ArweaveService _arweave;
   final DriveDao _driveDao;
   final ProfileCubit _profileCubit;
-
+  final SyncCubit _syncCubit;
```
Comment (CDDelta) [inline lib/blocs/fs_entry_rename/fs_entry_rename_cubit.dart:24]: nit: line between `_syncCubit` and `_isRenamingFolder`.

File: lib/blocs/fs_entry_rename/fs_entry_rename_cubit.dart [hunk 2]
```diff
@@ -85,12 +87,20 @@ class FsEntryRenameCubit extends Cubit<FsEntryRenameState> {
               .folderById(driveId: driveId, folderId: folderId)
               .getSingle();
           folder = folder.copyWith(name: newName, lastUpdated: DateTime.now());
-
           final folderTx = await _arweave.prepareEntityTx(
               folder.asEntity(), profile.wallet, driveKey);
 
           await _arweave.postTx(folderTx);
           await _driveDao.writeToFolder(folder);
+          var folderEntryCompanion = FolderEntriesCompanion.insert(
```
Comment (CDDelta) [inline lib/blocs/fs_entry_rename/fs_entry_rename_cubit.dart:95]: nit: use `final` as opposed to `var` whenever possible.

**Review by CDDelta — COMMENTED** [review]
Just some tiny nitpicks. LGTM! Awesome!

**Comment by CDDelta** [comment]
Looks like the preview deployment is failing because this PR is originating from a forked repo. Could you move your development onto this repo and create a PR from here instead?

**Comment by jdaev** [comment]
Alright

**Pull request closed without merging**